src/limb_repo/environments/lr_pybullet_env.py

```diff
@@ -0,0 +1,285 @@
+"""PyBullet environment for Limb Repositioning."""
+
+from dataclasses import dataclass
+
+import numpy as np
+import omegaconf
+from omegaconf import OmegaConf
+from scipy.spatial.transform import Rotation as R
+
+from limb_repo.environments.pybullet_env import (
+    PyBulletConfig,
+    PyBulletEnv,
+)
+from limb_repo.structs import BodyState, LRState, Pose
+from limb_repo.utils import pybullet_utils
+
+
+@dataclass
+class LRPyBulletConfig:
+    """Configuration for a Limb Repo PyBullet environment."""
+
+    pybullet_config: PyBulletConfig
+    active_base_pose: Pose
+    active_q: np.ndarray
+    active_urdf: str
+    passive_base_pose: Pose
+    passive_q: np.ndarray
+    passive_urdf: str
+    wheelchair_pose: Pose
+    # wheelchair_config: np.ndarray
+    wheelchair_urdf: str
+    active_ee_to_passive_ee: np.ndarray
+
+
+class LRPyBulletEnv(PyBulletEnv):
+    """Pybullet environment for Limb Repositioning."""
+
+    def __init__(self, config: omegaconf.DictConfig) -> None:
+        self.config = config
+
+        ## Initialize empty pybullet simulation
+        super().__init__(config.pybullet_config)
+
+        ## Set initial values
+        print("config active urdf", self.config.keys())
+        self.active_urdf: str = self.config.active_urdf
```

Comment:
it's generally good practice to make attributes private (e.g., `self._active_urdf`) unless you need external things to access them